src/Service/ClientMetadataService.php

```diff
@@ -45,6 +51,7 @@ public function generateMetadata(Request $request): array
             'request_id' => $this->requestId->get(),
             'user_agent' => $userAgent,
             'ip_address' => $ipAddress,
+            'sari' => $this->stateHandler->hasRequestId() ? $this->stateHandler->getRequestId() : null
```

Comment:
Did you compare this logic to, for example how Tiqr is retrieving the request ID? 
I'm not sure we explicitly log the 'sari' on the actual user-facing error, but log it in the extra section of the log entry.